feat: search node management with `mongodbatlas_search_deployment` resource and data source (#1633)
mongodb/terraform-provider-mongodbatlas: examples/atlas-search-deployment/README.md

# MongoDB Atlas Provider - Atlas Cluster with dedicated Search Nodes Deployment

> This example shows how you can use Atlas Dedicated Search Nodes in Terraform. As a prerequisite, you need to create a project and cluster resource.

Reviewer: this feels like the prerequisite has to be done separately instead it's part of the example
```suggestion
This example shows how you can use Atlas Dedicated Search Nodes in Terraform. As part of it, a project and cluster resource are created as a prerequisite.
```
